### ISSUE TITLE
LMILAB

### ISSUE THREAD

**Issue by jdbancal** [replab/replab#50]
One of the doc page (https://replab.github.io/replab/docs/publish/SDP.html) shows the following error

```
#############################################################
You are using LMILAB. Please don't use LMILAB with YALMIP
https://yalmip.github.io/solver/lmilab/
 
Install a better SDP solver
https://yalmip.github.io/allsolvers/
 
To get rid of this message, edit calllmilab.m  
(but don't expect support when things do not work,
YALMIP + LMILAB => No support)
#############################################################
```

Any idea which change could have caused that?

**Comment by denisrosset** [comment]
This error happens when no specific solver is in the path. I guess I forgot to run `install_sdpt3` when generating the docs.

**Comment by jdbancal** [comment]
I see, that could be. Personally, I get the following error if I don't run `install_sdp3`:
```
solvertime = 0
    info = No suitable solver
    problem = -2
    yalmiptime =  0.073166
```
But maybe you have an installation of LMILAB on your system. I'll add a check for sedumi in `replab_compiledoc`.

**Comment by denisrosset** [comment]
(`SDPT3` I guess)

**Comment by jdbancal** [comment]
oh, yes of course, haha!

**Comment by jdbancal** [comment]
Solved by #51
